tests/annotator/test_date_annotator.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+from __future__ import absolute_import
+import unittest
+import datetime
+from epitator.annotator import AnnoDoc
+from epitator.date_annotator import DateAnnotator
+
+
+class DateAnnotatorTest(unittest.TestCase):
```

Comment:
Should we add a test for dates with dashes (or slashes etc.)?  I'm seeing some weird behavior - I added the following test:

```    
def test_dashes(self):
        text = 'Adenoviruses, first seen between 2010-1-1 and 2010-1-2'
        doc = AnnoDoc(text)
        doc.add_tier(self.annotator)
        self.assertEqual(
            doc.tiers['dates'].spans[0].datetime_range,
            [datetime.datetime(2010, 1, 1),
            datetime.datetime(2010, 1, 2)])
```

and i'm getting the results:

```
AssertionError: Lists differ: [datetime.datetime(2010, 1, 2,... != [datetime.datetime(2010, 1, 1,...

First differing element 0:
2010-01-02 00:00:00
2010-01-01 00:00:00

- [datetime.datetime(2010, 1, 2, 0, 0), datetime.datetime(2010, 1, 3, 0, 0)]
?                             ^                                    ^

+ [datetime.datetime(2010, 1, 1, 0, 0), datetime.datetime(2010, 1, 2, 0, 0)]
?                             ^                                    ^
```